data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_15.xml.ep

```diff
@@ -3,7 +3,7 @@
 <profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
   <suse_register>
     <do_registration config:type="boolean">true</do_registration>
-    <reg_server>{{SCC_URL}}</reg_server>
+    <reg_server><%= defined $bmwqemu::vars{"HOST_SCC_URL"} ? $bmwqemu::vars{"HOST_SCC_URL"} : "" %></reg_server>
```

Comment:
I didn't check syntax details. Will `<reg_server>{{HOST_SCC_URL}}}</reg_server>`  work?

Reply:
No. I have tried that `{{HOST_SCC_URL}}` did not work. It seems that some settings, such as `SCC_URL` and `SCC_ADDONS`, are more like inline variables, they are supported in autoyast .xml.ep file.